### DISCUSSION TITLE
Is there any plan to support harmony

### DISCUSSION THREAD

**Discussion by mny459** [simolus3/drift#3169] [Q&A]
Great work first, Thank You!
I want to know is there any plan to support hormony. Which supported flutter is on https://gitee.com/openharmony-sig/flutter_flutter by community.

**Comment by simolus3** [comment]
`drift` is a Dart package that doesn't depend on Flutter or any native Flutter libraries directly. So it will work on any operating system on which Dart can run, including harmony if someone made the work to port it.

`sqlite3_flutter_libs` would be the package that needs work to support harmony, but if you find another way to install a bundled sqlite3 into your application, you don't even need that package. The examples on the repository you've posted don't seem to contain any harmony-specific native code and I also don't have devices to test this on, so I don't really have a way to get started with this or maintain it.
Welcomes to sqlite3\_flutter\_libs to add support for harmony are welcome if they don't break anything else or cause considerable maintenance overhead.

**Reply by mny459** [reply]
Got it. Thank you for your reply.
